tasks/get_current_archive_list/get_current_archive_list.py

```diff
@@ -66,8 +62,6 @@ def task(
     report_bucket_region: str,
     report_bucket_name: str,
     manifest_key: str,
-    s3_access_key: str,
-    s3_secret_key: str,
```

Comment:
I don't think graphql is using this code under tasks/get_current_archive_list/get_current_archive_list.py.

It uses https://github.com/nasa/cumulus-orca/blob/develop/graphql/src/adapters/graphql/resolvers/internal_reconcile_report.py#L58

Reply:
We have a ticket https://bugs.earthdata.nasa.gov/browse/ORCA-796 to remove these

Reply:
Sounds good, reverted those changes